react/components/__tests__/PickupPointDetails.test.js

```diff
@@ -488,4 +547,55 @@ describe('PickupPointDetails', () => {
 
     expect(handleClosePickupPointsModal.mock.calls).toHaveLength(1)
   })
+
+  it('should show the right info about the selected pickup point', () => {
+    const { queryByText, getByTestId } = render(
+      <ModalState
+        activePickupPoint={undefined}
+        address={address}
+        residentialAddress={undefined}
+        askForGeolocation={false}
+        googleMapsKey={'1234'}
+        isSearching={false}
+        items={state.orderForm.items}
+        mapStatus={'HIDE_MAP'}
+        logisticsInfo={modalState.logisticsInfo}
+        pickupPoints={modalState.pickupPoints}
+        pickupOptions={modalState.pickupOptions}
+        salesChannel={undefined}
+        orderFormId={undefined}
+        selectedPickupPoint={modalState.selectedPickupPoint}>
+        <IntlProvider
+          locale="pt"
+          messages={{ ...messages, ...{ 'country.BRA': 'BRA' } }}>
+          <PickupPointDetails {...props} />
+        </IntlProvider>
+      </ModalState>
+    )
```

Comment:
Just curious - why this is so different from than above tests? There is no way to wrap inside the provider?

Reply:
Actually the provider is already wrapped up by the ModalState component. I had to do like that because functions, such as `selectNextPickupPoint`, are crucial for the test to be efficient and we shouldn't mock it. So the logical thing to do was to move the test up in the tree.

Reply:
Makes sense! ok